### PR TITLE
Image creation script for Ubuntu 20.04

### PR DESCRIPTION
**Pull request by khabiruddin**
This script will build an image for Ubuntu20.04.

CCCP-3301

### CODE REVIEW

**Review by junousi — CHANGES_REQUESTED** [review]
Script looks good. Crontab in same directory needs modifications, too

**Comment by khabiruddin** [comment]
> Script looks good. Crontab in same directory needs modifications, too

Yes made crontab changes. This time only for cPouta devel and production before ePouta.

File: scripts/crontab [hunk 1]
```diff
@@ -12,6 +12,7 @@ SHELL=/bin/bash
 0 6 1,16 * * source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_ubuntu-16.04-cuda.sh >> $HOME/log/c/ubuntu-16.04-cuda.log 2>&1
 
 0 7 1,16 * * source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_ubuntu-18.04.sh >> $HOME/log/c/ubuntu-18.04.log 2>&1
+0 7 1,16 * * source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_ubuntu-20.04.sh >> $HOME/log/c/ubuntu-20.04.log 2>&1
```
Comment (junousi) [inline scripts/crontab]: You shouldn't do two builds at the same time, it can screw up the loop mount system

**Review by junousi — CHANGES_REQUESTED** [review]

File: scripts/crontab (hunk 1 above)
Comment (khabiruddin) [reply scripts/crontab]: Now each imagebuild at different time.

Comment (junousi) [reply scripts/crontab]:
```
0 10 * * * [...] >> $HOME/log/c-devel/centos7-cuda.log 2>&1
0 10 1,16 * * [...] >> $HOME/log/c/ubuntu-20.04.log 2>&1
```

These are still overlapping at 10AM on the 1st and 16th day of the month.

I think you need to push one or more of the cpouta-devel builds closer to evening to make this fit.

**Comment by junousi** [comment]
Also! You could already add the ePouta-prod entry in #commented form

File: scripts/crontab (hunk 1 above)
Comment (khabiruddin) [reply scripts/crontab]: Yes made another change for ubuntu-20.04. Now this is 30 10 1,6 * *.

**Comment by khabiruddin** [comment]
> Also! You could already add the ePouta-prod entry in #commented form

Yes Ubuntu20.04 build for ePouta in commented for just for now.

**Comment by khabiruddin** [comment]
Now I have given 1hour gaps between image build.

**Comment by junousi** [comment]
It's a bit strange order not having them in a continuous range, but OTOH it's more outside 9-to-5 office hours or 7-to-19 SLA hours. Works for me.

**Review by junousi — APPROVED** [review]

**Pull request merged**